src/iitkgp_erp_login/errors.py

```diff
@@ -0,0 +1,4 @@
+ANSWER_MISSMATCH_ERROR = "Unable to send OTP due to security question's answare mismatch ."
+PASSWORD_MISSMATCH_ERROR = "Unable to send OTP due to password mismatch."
+OTP_SENT_MESSAGE = "An OTP(valid for a short time) has been sent to your email id registered with ERP, IIT Kharagpur. Please use that OTP for further processing. "
+OTP_MISMATCH_ERROR = "ERROR:Email OTP mismatch"
```

Comment:
Are thesse error sent by erp? looks like it since there are many typos :)

Reply:
Yes 🤣